manifests/service_reload.pp

```diff
@@ -1,6 +1,8 @@
-# This class implements prometheus service reload
-# without restarting the whole service when a config
-# changes
+#
+# @summary
+#   This class implements prometheus service reload without
+#   restarting the whole service when a config changes
+#
```

Comment:
I think this class *is* definately `@api private` though.

Reply:
fixed it